src/components/ProductItem/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Fragment } from 'react';
+import { ListIcon, ListItem, Text } from '@chakra-ui/react';
+import { FaPlus } from 'react-icons/fa';
+
+
+type ProductItemProps = {
+    items: Array<string>;
+}
+
+export const ProductItem = ({ items }: ProductItemProps) => {
+    return (
+        <Fragment>
+            {items.map((item, index) => (
+                <ListItem
+                    key={index}
```

Comment:
Index is a bad key for a List on React.

[Link to offical doc reference]( https://legacy.reactjs.org/docs/lists-and-keys.html#:~:text=We%20don't%20recommend%20using,cause%20issues%20with%20component%20state.)

Create a `id` field on mock object is better approach